src/graph.py

```diff
@@ -752,6 +741,105 @@ def perform_GO_enrichment_on_clusters(nxgraph, clustering):
     """
     pass
 
+
+###############################################################################
+            #######            Results           #######
+###############################################################################
```

Comment:
I think these functions should be under the **grid search** heading, no?

Reply:
Like as an attribute of the class? I considered this, but none of those functions require `self`. Would that be poor form? 

Another option is to run these functions automatically following the randomizations, but it's nice for the user to be able to play with parameters in `get_robust_subgraph_from_randomizations` to generate their own robust networks.